src/screens/SignUp.js

```diff
@@ -0,0 +1,120 @@
+import React, { Component } from "react";
+import { NavLink, } from "react-router-dom";
+import { validateEmail, validateName, validatePassword } from "../assets/helpers/userValidationTools"
+
+
+class SignUp extends Component {
+   constructor(props) {
+      super(props);
+
+      this.state = {
+         firstName: '',
+         lastName: '',
+         email: '',
+         password: '',
+         passwordVerification: '',
+         formIsValid: true,
```

Comment:
We should default this to false just to make things extra secure

Reply:
done